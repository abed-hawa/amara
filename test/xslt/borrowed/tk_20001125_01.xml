<?xml version='1.0' encoding='iso-8859-1'?>
<doc>
<para>H\344agen-Dazs</para>
<note>H\344agen-Dazs</note>
</doc>