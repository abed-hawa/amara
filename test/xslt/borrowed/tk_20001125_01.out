<?xml version='1.0' encoding='iso-8859-1'?>\012\012<para>H\344agen-Dazs</para>\012<!--H\344agen-Dazs-->\012